CMakeLists.txt

```diff
@@ -67,8 +67,15 @@ include(cmake/SetupPackages.cmake)
 include(cmake/SetupRajaConfig.cmake)
 
 # Configure a header file with all the variables we found.
+get_directory_property( RAJA_DEFINITIONS_INITIAL DIRECTORY ${CMAKE_SOURCE_DIR} COMPILE_DEFINITIONS )
+set(RAJA_DEFINITIONS "")
+foreach(definition ${RAJA_DEFINITIONS_INITIAL})
+  set(RAJA_DEFINITIONS "${RAJA_DEFINITIONS} -D${definition}")
+endforeach()
```

Comment:
If anybody knows a cleaner way to do this (pick up all of the definitions passed to the compiler) I'd like to use that, but this was the best I could do on short notice